### DISCUSSION TITLE
Headless mode works fine on one system but getting "Sorry you have been blocked" on another system

### DISCUSSION THREAD

**Discussion by tooti12** [seleniumbase/SeleniumBase#2725] [Q&A] [answered]
I'm facing an issue when browsing using headless2 mode on one of my systems. I have checked the chrome version , window size, the headers and everything is exactly same on both the laptops but when I run the script on one of them I get "Sorry you have been blocked" respone.

**Comment by mdmintz** [comment, marked as answer]
Sounds like they already blocked that system's IP Address for unusual web traffic. If the IP Address wasn't actually blocked, you may be able to use `xvfb=True` instead of `headless2=True`. I can't be of much help without more actual details of what you're doing.

**Reply by tooti12** [reply]
Thanks for the update @mdmintz . I will try this. I'm definitely sure the systems IP address is not blocked since I also have a proxy. If the ```xvfb=True``` value works then all good otherwise I'll provide you with the code snippet and screenshots in detail of what I'm trying to do.
